packages/api/src/store/db.ts

```diff
@@ -195,36 +205,44 @@ export class DB {
     query: string | QueryConfig<I>,
     values?: I
   ): Promise<QueryResult<T>> {
-    let queryLog;
+    let queryLog: string;
+    const labels: any = {};
```

Comment:
**nit:** Perhaps a more accurate type.

```suggestion
    const labels: Record<string, string> = {};
```

We should avoid `any` at all costs since that is just regressing to ~~savagery~~ JavaScript :laughing: